### PR TITLE
DYN-3771 Update verbiage for the Linter View Extension help docs

### PR DESCRIPTION
**Pull request by saintentropy**
### Purpose

This tasks updates the documentation content for the Linter View extension help docs.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@nate-peters @QilongTang 

### FYIs

@mjkkirschner @BogdanZavu

### CODE REVIEW

**Comment by QilongTang** [comment]
Good improvement to me. Recently our team begin to involve Content designer Helena to help us review materials. I am only curious if you guys are doing similar approach?

File: src/LintingViewExtension/Docs/LinterViewHelpDoc.html [hunk 1]
```diff
@@ -174,12 +174,15 @@
...
+    <h4>Node Issues and Graph Issues</h4>
+    <p>Rules defined by an extension are registered as one of two types: Graph rules, or Node rules.  If Dynamo detects that one of those rules has been broken in the current graph, a warning with information on how to fix the issue, and a link to where the issue was detected will be shown in one of these two sections.</p>
+    <h4>Other Info</h4>
+    <p>Whenever a graph is saved, the Graph Status extension will store some metadata in the Dynamo file such as the selected Graph Type, and whether or not issues were present when the graph was last saved.</p>
+    <p>When a new Dynamo graph is created, the Graph Status extension doesn't check for any specific rules automatically. The default selection for the Graph Type will be <i>None</i>. Currently, only one Graph Type can be selected at a time in the Graph Status view.</p>
```
Comment (QilongTang) [inline src/LintingViewExtension/Docs/LinterViewHelpDoc.html:187]: Do you mean graph open for graph created or both?

**Review by QilongTang — APPROVED** [review]

File: src/LintingViewExtension/Docs/LinterViewHelpDoc.html (hunk 1 above)
Comment (saintentropy) [reply src/LintingViewExtension/Docs/LinterViewHelpDoc.html:187]: both... will update

Comment (saintentropy) [reply src/LintingViewExtension/Docs/LinterViewHelpDoc.html:187]: oops this is only for created...  Open will depend on how the graph was saved by the user.

**Comment by saintentropy** [comment]
@QilongTang I don't know if Nate worked with Helena.  I will check with him when he returns from vacation.

**Pull request merged**